.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+## Feature Status
+**READY/IN DEVELOPMENT**
+
+## Related PRs
+List related PRs against other branches e.g. for backporting features/bugfixes
+to previous release branches:
+
+Repo/Branch | PR
+------ | ------
+some_other_PR | [link]()
+
+
+## Todos
+- [ ] Tested and working on development environment
+- [ ] Unit tests (if appropriate)
+- [ ] New API endpoints documented. Add [link]() if different from this PR 
+- [ ] Reviewed and approved by a senior dev
```

Comment:
We don't really have a concept of senior devs. Anyway, GitHub now has the built-in list of reviewers so this line can probably be removed.